R/jm_data_block.R

```diff
@@ -171,7 +171,7 @@ reformulate_rhs <- function(x, subbars = FALSE) {
 handle_cov_prior <- function(prior, cnms, ok_dists = nlist("decov", "lkj")) {
   if (!is.list(prior)) 
     stop(sQuote(deparse(substitute(prior))), " should be a named list")
-  t <- length(unique(cnms)) # num grouping factors
+  t <- length(cnms)         # num grouping factors
```

Comment:
Anyone remember why this changed from `t <- length(unique(cnms))` to `t <- length(cnms)`. I'm having trouble imagining a situation where `cnms` would have duplicates but a harder time imagining a situation when we would want to count the duplicates.

Reply:
I can't remember why I changed this -- but I obviously thought the previous form was a bug: https://github.com/stan-dev/rstanarm/commit/b735546d5172447995e1a0bbec1de2718feb02ee#diff-25a3bdc58900c361b6e3d7f6c892d124

Reply:
Obviously would have been something to do with the multivariate models and when two submodels have the same grouping factor not wanting to treat them as the same.